tendrl/monitoring_integration/flows/setup_cluster_alias/__init__.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+from tendrl.monitoring_integration.graphite import graphite_utils
+
+
+class SetupClusterAlias(flows.BaseFlow):
+
+    def run(self):
+        super(SetupClusterAlias, self).run()
+        integration_id = self.parameters["TendrlContext.integration_id"]
+        short_name = self.parameters["Cluster.short_name"]
+        alias_dir_path = "%s/name" % graphite_utils.get_data_dir_path()
+        if not os.path.exists(alias_dir_path):
+            try:
+                os.makedirs(str(alias_dir_path))
+            except OSError as ex:
+                raise FlowExecutionFailedError(
+                    "Failed to create cluster alias dir: (%s)"
+                    " .Error: (%s)" %
+                    (str(alias_dir_path), ex)
+                )
+        alias_path = "%s/%s" % (alias_dir_path, short_name)
```

Comment:
Where this var is used?

Reply:
Oh its not required as below its being created again. will remove this